packages/util/src/encoding.ts

```diff
@@ -1,4 +1,25 @@
 // Reference: https://ethereum.org/en/developers/docs/data-structures-and-encoding/patricia-merkle-trie/
+//
+// Trie keys are dealt with in three distinct encodings:
```

Comment:
Can possibly refactor over to using an enum for the three main encodings used in snap sync: KEYBYTES (0), HEX (1), COMPACT (2).